citi-onboarding/MeuApe-onepage
# fix/intro: fix background image in home and aline intro

## What I did:

- fix background image in home and aline intro

## How to test
1. Open the terminal and go to the folder of this project

~~~~typescript
git checkout develop
~~~~
~~~~typescript
git pull
~~~~
~~~~typescript
git checkout fix/intro
~~~~
~~~~typescript
git pull origin fix/intro
~~~~
~~~~typescript
yarn install
~~~~
~~~~typescript
yarn start
~~~~

2. In your browser go to <http://localhost:3000>

3. Check if the component looks just like the mockup in different screen sizes

4. Read the changed files section on GitHub

5. Test the things that you think are worth testing, even the ones that are not in this description
